site/docs/v1/tech/apis/_application-response-body-base.adoc

```diff
@@ -120,6 +120,15 @@ When this configuration is omitted, all HTTP origins are allowed to use the brow
 [field]#{base_field_name}.oauthConfiguration.authorizedRedirectURLs# [type]#[Array<String>]#::
 An array of URLs that are the authorized redirect URLs for FusionAuth OAuth.
 
+[field]#application.oauthConfiguration.clientAuthenticationPolicy# [type]#[String]# [since]#Available since 1.28.0#::
+Determines the client authentication requirements for the OAuth 2.0 Token endpoint.
++
+The possible values are:
++
+* `Required` - The client must provide client credentials when using the Token endpoint.
+* `NotRequired` - Providing client credentials is optional when using the Token endpoint.
+* `NotRequiredWhenUsingPKCE` - The client must provide client credentials when using the Token endpoint unless a valid PCKE `code_verifier` has been provided in the request body using POST data.
```

Comment:
If this is the same between several sections, it can be moved into an include. Or you can sync up the text to ensure the descriptions are the same between usages.